docs/configuration.md

```diff
@@ -1040,10 +1040,9 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
   <td><code>spark.eventLog.compression.codec</code></td>
-  <td></td>
+  <td>zstd</td>
   <td>
-    The codec to compress logged events. If this is not given,
-    <code>spark.io.compression.codec</code> will be used.
+    The codec to compress logged events.
```

Comment:
sorry for coming in late, was out last week, we may want to reference what other codecs can be used here.   @dongjoon-hyun thoughts?

Reply:
Thank you for review, @tgravescs . Sure, I'll make a documentation follow-up.

Reply:
Here, I made a PR.
- https://github.com/apache/spark/pull/31695